changelogs/fragments/747-failed_when_the_job_name_was_null_or_not_found.yaml

```diff
@@ -0,0 +1,3 @@
+bugfix:
+- zos_job_output - Error message did not specifies the job not found. Fix now specifies the job_id or job_name searching for, to ensure more information to the user.
```

Comment:
I'm sorry I did not saw the first sentence. This should be worded like :
"Error message did not specify the job not found. Fix now specifies the job_id or job_name searching for, to ensure more information to the user."